docs/software/exportformat.md

```diff
@@ -61,10 +62,35 @@ are still included in reporting to provide access to historic data. Deprecated c
 | Referring Tool                           | For users who were linked to the absentee tool by another tool, the name of the tool                                                                                                                                                                 |
 | Updated At (UTC)                         | The date and time when this action's data was last updated, in UTC                                                                                                                                                                                   |
 
+## Calendar tool export format
 
-## FutureVoter Tool Export Format
+| Column title          | Description                                                                                |
+|-----------------------|--------------------------------------------------------------------------------------------|
+| ID                    | The ID we store internally for the verification                                            |
+| Subscriber            | The name of the subscriber                                                                 |
+| Time Started (UTC)    | The time that the first page of the verification was done                                  |
+| Email                 | The email address of the user                                                              |
+| Phone                 | The phone number of the user                                                               |
+| Address 1             | The user's address1                                                                        |
+| Address 2             | The user's address2                                                                        |
+| City                  | The user's city                                                                            |
+| Zipcode               | The user's zipcode                                                                         |
```

Comment:
So this is the format for the Calendar export - and yeah we do store these if we are able to parse them out from the address.

For FutureVoter - No, we don't even have address 1, address 2 or city on the model. Technically, zipcode is there because we had debated asking for zip instead of state, but of course it's not being populated currently.

Reply:
Sorry I completely put this in the wrong place. It was solely meant for futurevoter! Thanks for explaining; I think what I was thinking was that we might want register actions (data is saved) that are referred to futurevoter to also save the data in the futurevoter action. But I now remember we already have a ticket about saving PII in general, so we should just move on. Thanks for bearing with me!